app/controllers/spaces_controller.rb

```diff
@@ -35,6 +36,10 @@ def destroy
     redirect_to spaces_path
   end
 
+  def my_spaces
+    # TODO: show personal spaces in own view; ability to confirm reservation inline
```

Comment:
Yep, this one is needed (at least the view)

Reply:
@Actionmaker1